health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/Boundary.js

```
@@ -0,0 +1,210 @@
+                            label={t("CREATE_MY_OWN_BOUNDARY_DATA")}
+                            onClick={() => {
+                                callGeoPode(true);
+                            }}
+                        />
+                    </div>
+                    </PopUp>
+
+                )}
+                <Card type={"primary"} variant={"viewcard"} className={"example-view-card"}>
+                    <div style={{fontWeight:700}}>Boundary Data Management</div>
+                    <div style={{height:"2rem"}}></div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)